src/index.js

```diff
@@ -132,7 +133,7 @@ class PeerId {
   }
 }
 
-exports = module.exports = PeerId
+exports = module.exports = withIs(PeerId, { className: 'PeerId', symbolName: '@libp2p/js-peer-id/PeerId' })
```

Comment:
There are a number of occasions in this file where we're accessing the non-wrapped `PeerId` class:

https://github.com/libp2p/js-peer-id/blob/6513a02e1e80ea38da5ee3520d1ab4c6c9432e02/src/index.js#L157
https://github.com/libp2p/js-peer-id/blob/6513a02e1e80ea38da5ee3520d1ab4c6c9432e02/src/index.js#L162
https://github.com/libp2p/js-peer-id/blob/6513a02e1e80ea38da5ee3520d1ab4c6c9432e02/src/index.js#L166
https://github.com/libp2p/js-peer-id/blob/6513a02e1e80ea38da5ee3520d1ab4c6c9432e02/src/index.js#L170
https://github.com/libp2p/js-peer-id/blob/6513a02e1e80ea38da5ee3520d1ab4c6c9432e02/src/index.js#L199
https://github.com/libp2p/js-peer-id/blob/6513a02e1e80ea38da5ee3520d1ab4c6c9432e02/src/index.js#L231
https://github.com/libp2p/js-peer-id/blob/6513a02e1e80ea38da5ee3520d1ab4c6c9432e02/src/index.js#L282

These all need to be changed otherwise they create `PeerId` objects that don't have the '@libp2p/js-peer-id/PeerId' symbol and so aren't considered to be instances of `PeerId` when used with `PeerId.isPeerId`.

Reply:
Yes! Thanks for noticing that.

I added `PeerIdWithIs` to all of them. However, I feel that there might be a better approach here. What do you think?

Reply:
Not really, js-cid uses `_CID` https://github.com/ipld/js-cid/blob/6d249ab8a3cc4000a1db0467f23046650453049a/src/index.js#L243